pkg/multiplexer/tripper.go

```diff
@@ -0,0 +1,26 @@
+package multiplexer
+
+import "net/http"
+
+type Tripper interface {
+	Request(r *http.Request) (*http.Response, error)
+}
+
+type StandardTripper struct {
+	client *http.Client
+}
+
+// Creates a new instance of a Tripper. The passed in client is modified to
+// have no cookie jar and to not follow redirects.
+func NewStandardTripper(client *http.Client) *StandardTripper {
```

Comment:
Perhaps this should just return `Tripper` and then `StandardTripper` could be private since it has only internal fields at this point.